se/spelling.py

```diff
@@ -374,6 +374,8 @@ def modernize_spelling(xhtml: str) -> str:
 	xhtml = regex.sub(r"incumber", r"encumber", xhtml)				# incumber -> encumber
 	xhtml = regex.sub(r"\bCadi\b", r"Qadi", xhtml)					# cadi -> qadi
 	xhtml = regex.sub(r"\bcadi\b", r"qadi", xhtml)					# cadi -> qadi
+	xhtml = regex.sub(r"\bSoldan\b", r"Sultan", xhtml)					# soldan -> sultan
```

Comment:
We should omit the trailing \b, otherwise we won't catch `soldans` (plural)

Reply:
Or, change the regex to `\bsoldans?\b` which would probably be best

Reply:
Changed to `\bsoldans?\b` and `\bSoldans?\b`